provider/oauth2/tests.py

```diff
@@ -126,6 +126,20 @@ def test_authorization_requires_supported_response_type(self):
         response = self.client.get(self.auth_url2())
         self.assertEqual(200, response.status_code)
 
+    def test_token_authorization_redirects_to_correct_uri(self):
+        self.login()
+
+        response1 = self.client.get(self.auth_url() + '?client_id=%s&response_type=token' % self.get_client().client_id, follow=True)
```

Comment:
@justinabrahms ah, you updated this just as I was going to respond :-)

In general we try to stick with a mindset of improving/updating code style where possible when adding/changing stuff.  That does sometimes conflict with the notion of maintaining internal consistency as you've pointed out, and also, this being a forked repo, it's even more of a special case.  So it wouldn't have been a dealbreaker, but appreciate your making the changes.